### DISCUSSION TITLE
quarkus-maven-plugin - how to ignore resource files being copied to the fast jar

### DISCUSSION THREAD

**Discussion by jaiprasad266** [quarkusio/quarkus#26000] [Q&A]
Hi , 
As per the documentation fast-jar is the default quarkus package type.
Now I want to exclude certain resource files and classes from the jar.

This is the property used.

quarkus.package.type=jar
quarkus.package.user-configured-ignored-entries=src/main/resources/sql.json

However the sql.json is present in the jar.

Tried building the uber jar and excluding files and it works , by using below property.
quarkus.package.type=uber-jar
quarkus.package.ignored-entries=src/main/resources/sql.json

But how to the same with the fast-jar ? 

Reference Links
https://quarkus.io/guides/maven-tooling#fast-jar
https://quarkus.io/guides/maven-tooling#uber-jar-maven

[a comment by an automated account is not shown]

**Comment by rakeshramesh1404** [comment]
Hello after 3 years,
Is this fixed in the later versions bcz I still face the above mentioned problem in 3.26.1?
